sdks/java/extensions/sql/build.gradle

```diff
@@ -103,9 +103,9 @@ shadowJar {
     include(dependency("org.codehaus.janino:.*"))
     include(dependency(library.java.protobuf_java))
   }
-  relocate "com.google.protobuf", "org.apache.beam." + project.name.replace("-", ".") + "repackaged.com.google.protobuf"
```

Comment:
Don't remove the `beam.`, it seems like the missing `.` infront of repackaged is all we want.

Reply:
done

Reply:
added comment to  the `org.codehaus` relocation config